### PR TITLE
feat: replace azure-wheel-helperes with cibuildwheel

### PR DESCRIPTION
**Pull request by henryiii**

### CODE REVIEW

File: _data/projects/misc/cibuildwheel.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  [See our guide for instructions](https://scikit-hep.org/developer/gha_wheels).
+url: https://github.com/joerick/cibuildwheel
+readme: https://github.com/joerick/cibuildwheel/blob/master/README.md
+docs: https://cibuildwheel.readthedocs.io
+affiliated: true
+longdescription: |
+
+  The [cibuildwheel](https://cibuildwheel.readthedocs.io) project provides a Python package that can be installed on CI systems to build Python wheels. All wheels possible for an OS are built, using best practices for installing Python for building compatibly. Extensive environment variable configuration allow exact version specification, manylinux wheel selection, tests, and much more. Specialty architectures and PyPy are also supported.
+  
+  This replaced the [azure-wheel-helpers]()https://github.com/scikit-hep/azure-wheel-helpers repository that was previously used to build wheels for Scikit-HEP. For up-to-date instructions, check the [Scikit-HEP developer pages](https://scikit-hep.org/developer/gha_wheels).
```
Comment (eduardo-rodrigues) [inline _data/projects/misc/cibuildwheel.yml:13]:
```suggestion
  This replaces the [azure-wheel-helpers](https://github.com/scikit-hep/azure-wheel-helpers) repository that was previously used to build wheels for Scikit-HEP. For up-to-date instructions, check the [Scikit-HEP developer pages](https://scikit-hep.org/developer/gha_wheels).
```

File: pages/packages/affiliated.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ parent: Packages
 Affiliated projects and packages {#affiliated_packages}
 ================================
 
-In the following, projects that work closely together with Scikit-HEP are described. They extend the Python ecosystem in HEP and remain, due to their size and scope, generally independent of Scikit-HEP.
+In the following, projects that work closely together with Scikit-HEP are described. They extend the Python ecosystem in HEP and remain, due to their size and scope, generally independent of Scikit-HEP. In most cases, at least one Scikit-HEP core developer is on the maintainer team of these packages. These packages are highly recommended for Scikit-HEP developers and users.
```
Comment (eduardo-rodrigues) [inline pages/packages/affiliated.md:13]:
```suggestion
In the following, projects that work closely together with Scikit-HEP are described. They extend the Python ecosystem in HEP and remain, due to their size and scope, generally independent of Scikit-HEP. In most cases, at least one Scikit-HEP core developer is on the maintainer team of, or regularly contributes to, these packages. These packages are highly recommended for Scikit-HEP developers and users.
```

**Review by eduardo-rodrigues — APPROVED** [review]
I made a couple of suggestions. Let me ping @mayou36 since I believe he made comments during our discussion on the admins list ...

**Comment by mayou36** [comment]
Thanks, this looks good to me; vaguely enough to not exclude someone but clear enough to state the intentions

**Comment by henryiii** [comment]
At least @jpivarski had comments too. Though I think this follows those by remaining non-absolute in requirements.

**Pull request merged**